ci/version_var.sh

```diff
@@ -11,7 +11,9 @@ BASE_VERSION=0.1.5
 IS_RELEASE=false
 
 SOURCE_REPO=edge-agent
-BASE_USER_WASM_PKG_NAME=user-agent-wasm
+BASE_USER_AGENT_PKG_NAME=user-agent
+BASE_USER_AGENT_SUPPORT_PKG_NAME=user-agent-support
```

Comment:
IMO web-wallet/agent and web-wallet/agent-support would be more appropriate names for the images, as these are just supporting agent features on browser/web.

@troyronda @fqutishat @llorllale @soluchok thoughts ?

Reply:
Other wallet types also need recovery via login. It's not clear that _all_ the features are limited to a Web Wallet.

Reply:
I do think user-agent is a confusing name for a backend service.